wsgiservice_restplus/namespace.py

```diff
@@ -192,6 +192,60 @@ def expect(self, *inputs, **kwargs):
         return self.doc(**params)
 
 
+    def payload_model(self, *models):
+        """A decorator that adds payload parameters model data to swagger api documentation as well as
+        applies wsgiservice validation using the model object provided.
+
+        :param models: model(s) objects to extract api information from on request payload model
+        """
+
+        expect = []
+        params = {
+            'validate': self._validate,
+            'expect': expect
+        }
+        validations = {}
+
+        for model in models:
+            expect.append(model)
+            model_validations = self._prepare_validation_dict(model)
+            validations.update(model_validations)
+
+        def wrapper(documented):
+
+            if not hasattr(documented, '_validations'):
+                documented._validations = {}
+            documented._validations.update(validations)
+
+            self._handle_api_doc(documented, params)
+            return documented
+
+        return wrapper
+
+
+    def _prepare_validation_dict(self, model):
+
+        """Generates the content of the _validations dictionary (normally used by validate decorator \
+        from wsgiservice.decorators) from a single Model object.
+
+        :param model: instance of the
+        :return: validations dictionary (equivalent to _validations from validate decorator from wsgiservice)
+        :rtype: dict
+        """
+
+        validations = {}
+
+        for field_name, field in model.items():
```

Comment:
Isn't `iteritems()` better? As far as I know items build a whole list before returning. Not a big optimiziation, rather a best practice.